README.md

```diff
@@ -5,6 +5,10 @@
 If you work on more than one feature at a time, you are guaranteed to multiply your bugs and your anxiety.
 
 ## Making a plan
+1. Add my name & pronouns
```

Comment:
Great job with your plan! Something that might also be helpful to add is what sorts of HTML elements you might need for each of your steps.